mkosi/__init__.py

```diff
@@ -176,6 +178,18 @@ def remove_packages(state: MkosiState) -> None:
             die(f"Removing packages is not supported for {state.config.distribution}")
 
 
+def check_none_root_populated(state: MkosiState) -> None:
+    """If no distribution was installed, check that root was populated by looking for os-release file."""
+    if state.config.distribution != Distribution.none:
+        return
+
+    osrelease = state.root / "usr/lib/os-release"
+    if osrelease.is_file():
+        return
+
+    die("Root must be populated from skeleton tree or by other means for distribution 'none'")
```

Comment:
I think it'd be better to say:
```python
die(f"{osrelease} is not a file.",
    hint=""Root must be populated from skeleton tree or by other means for distribution 'none'")
```

Reply:
Thanks, combined with the request to make it distribution-agnostic